### ISSUE TITLE
Fix docs and typescript types

### ISSUE THREAD

**Issue by HironTez** [actifyjs/actify#27]
**Describe the bug**
Please fix the docs. The docs is often missleading. For example it says there's a property "disabled" in the TextField component even though it's actually "isDisabled".

Also i would never know that you should use "onPress" instead of "onClick" if i haven't seen the warning from react-aria in the browsers console. The onClick doesn't work on mobile.

And there's a plenty of properties missing in the docs, like Checkbox "variant".

Please fix the typescript types. Some properties are missing in types, like "label" in TextField. Types of some components just don't work, for example types of Checkbox and Button

**Desktop (please complete the following information):**
 - OS: Ubuntu 24
 - Browser Chrome, Firefox
 - Version lates

**Smartphone (please complete the following information):**
 - OS: Android 15
 - Browser Chrome Android, Firefox Android

**Additional context**

**Comment by HironTez** [comment]
Upd: it's very strange but all the types have been fixed once i installed "@react-spectrum/toast"

**Comment by HironTez** [comment]
I've checked it multiple times and types don't work correctry without react-spectrum. As a temporary fix i've added it to the devDependencies in my package.json.

**Comment by lerte** [comment]
You are right, I am currently refactoring my components to use react-aria.
So the documentation is not up to date.
Thanks for your feedback.
